scrapers/lethalpass.yml

```diff
@@ -0,0 +1,29 @@
+name: lethalpass
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lethalpass.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@class="title"]/h1
+     Date:
+        selector: //div[@class="views"]/span
+        postProcess:
+          - replace:
+              - regex: "Added "
+                with: ""
+          - parseDate: 2 January 2006
+     Performers:
+      Name: //a[@class="model"]//a
+     Details: //div[@class="description"]/p
+     Tags:
+      Name: //a[@class='cat' and 1]
```

Comment:
what does the `and 1` accomplish? isnt it always true?

Reply:
Yup 1 can come out it is indeed always true.